site/content/docs/5.2/examples/features/index.html

```diff
@@ -285,4 +285,63 @@ <h4 class="fw-bold mb-0">Featured title</h4>
       </div>
     </div>
   </div>
+
+  <div class="b-example-divider"></div>
+
+  <div class="container px-4 py-5">
+    <h2 class="pb-2 border-bottom">Features with title</h2>
+
+    <div class="row row-cols-1 row-cols-md-2 align-items-md-center g-5 py-5">
+      <div class="d-flex flex-column align-items-start gap-2">
+        <h2 class="fw-bold">Left-aligned title explaining these awesome features</h2>
+        <p class="text-muted">Paragraph of text beneath the heading to explain the heading. We'll add onto it with another sentence and probably just keep going until we run out of words.</p>
+        <a href="#" class="btn btn-primary btn-lg">Primary button</a>
+      </div>
+      <div class="row row-cols-1 row-cols-sm-2 g-4">
+        <div class="d-flex flex-column gap-2">
+          <div
+            class="feature-icon-small d-inline-flex align-items-center justify-content-center text-bg-primary bg-gradient fs-4 rounded-3">
+            <svg class="bi" width="1em" height="1em">
+              <use xlink:href="#collection" />
+            </svg>
+          </div>
+          <h3 class="fw-semibold mb-0">Featured title</h3>
```

Comment:
in light of https://github.com/twbs/bootstrap/pull/36829 and keeping a more sensible heading structure, could you change this and all subsequent ones to `<h4>` ?